entries/p/pixel.labcorp.com.json

```diff
@@ -0,0 +1,13 @@
+{
+  "Pixel by Lapcorp": {
+    "domain": "pixel.labcorp.com",
+    "url": "https://www.pixel.labcorp.com",
```

Comment:
```suggestion
```
The `url` field isn't necessary when the site is accessible from https://`domain`